Src/library/vision2/interface/support/ev_monitor_dpi_detector.e

```diff
@@ -13,6 +13,12 @@ feature --Access
 			-- DPI sizes 96, 120, 144, 192, etc.
 		deferred
 		end
+
+	scaled_size (a_size: INTEGER): INTEGER
+			-- Return the scale size of input `a_size`.
```

Comment:
comment could be shorter, as   "-- Scaled size of `a_size`. "

Reply:
Done.